extensions/mentions/migrations/2022_10_21_000000_create_post_mentions_groups_table.php

```diff
@@ -15,6 +15,10 @@
     function (Blueprint $table) {
         $table->integer('post_id')->unsigned();
         $table->integer('mentions_group_id')->unsigned();
+        $table->timestamp('created_at')->nullable();
```

Comment:
```suggestion
        $table->timestamp('created_at')->useCurrent()->nullable();
```

Reply:
After this change, the tests fail on `MySQL` 5.7 - I think to do with the comment on dbal in the deleted migration.

@SychO9 - https://github.com/flarum/framework/actions/runs/3311921966/jobs/5467972384